src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -532,9 +532,12 @@ function ($p) {
      */
     private function mapStep2FormData(Product $product)
     {
+        // ecotax is stored with tax included but form uses the tax excluded value
+        $ecotax = \Tools::ps_round($product->ecotax * (1 + \Tax::getProductEcotaxRate() / 100), 2);
```

Comment:
Please declare `Tools` and `Tax`.

Reply:
But it seems magic !

Reply:
Nice fix! Once it's merged in 176 I think we should adapt it with `ComputingPrecision` for 177
Same as @Progi1984 As soon as the `use` statement I approve right away 😉

Reply:
PR updated. I found that in this class adapter classes were available so I used already declared adapter classes instead.